source/tests/pt/test_hybrid_muon.py

```
@@ -774,5 +774,345 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
+        """Single-signature checkpoints retain the uniform fast path."""
+        model, optimizer = self._make(graph_on=True)
+        self._run(model, optimizer, 3)
+        state_dict = optimizer.state_dict()
+        for group in state_dict["param_groups"]:
+            group["beta1_pow_device"] = torch.tensor(0.9**3, device=self.device)
+            group["beta2_pow_device"] = torch.tensor(0.95**3, device=self.device)
+        for state in state_dict["state"].values():
+            state.pop("bias_correction1", None)
+            state.pop("bias_correction2", None)
+
+        model2, optimizer2 = self._make(graph_on=True)
```

Ruff (0.16.2)

[warning] Unpacked variable `model2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)